clang/lib/Sema/SemaBounds.cpp

```diff
@@ -4017,11 +4023,55 @@ namespace {
         << ObservedBounds << ObservedBounds->getSourceRange();
     }
 
+    // BlameAssignmentWithinStmt prints a diagnostic message that highlights the
+    // assignment expression in St that causes V's observed bounds to be unknown
+    // or invalid.  If St is a DeclStmt, St itself and V are highlighted.
+    // BlameAssignmentWithinStmt returns the source location of the blamed
+    // assignment.
+    SourceLocation BlameAssignmentWithinStmt(Stmt *St, 
+                 const VarDecl *V, CheckingState State, unsigned DiagId) const {
```

Comment:
Please align `const VarDecl *V` underneath `Stmt *St` (and ensure the remaining parameters are on separate lines if necessary in order to prevent lines longer than 80 characters).